dockers/docker-orchagent/swssconfig.sh

```diff
@@ -38,7 +38,7 @@ fast_reboot
 
 HWSKU=`sonic-cfggen -m /etc/sonic/minigraph.xml -v minigraph_hwsku`
 
-SWSSCONFIG_ARGS="00-copp.config.json ipinip.json mirror.json "
+SWSSCONFIG_ARGS="00-copp.config.json ipinip.json mirror.json ports.json "
```

Comment:
if the port speed is configured here, what's the default port speed? what about adding port speed to port_config.ini file as a separate column?